app/main.py

```diff
@@ -2,0 +1,23 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command_split = command.split()
+    source_file_name = command_split[1]
+    directory = "/".join(command_split[2].split("/")[:-1])
```

Comment:
Consider using `os.path.join()` instead of string concatenation for paths. This will ensure cross-platform compatibility and make your code more robust.